src/ModbusAnalyzerSettings.cpp

```diff
@@ -182,6 +191,7 @@ const char* ModbusAnalyzerSettings::SaveSettings()
     // text_archive << mParity;
     // text_archive << mShiftOrder;
     text_archive << mInverted;
+    text_archive << mAssumeDataStartsWithResponse;
```

Comment:
This code is only used by Logic 1.x now, but it might be worth maintaining since there are users still on 1.x (mainly for the automation API, which isn't in Logic 2 yet)

SaveSettings and LoadSettings handle serializing the analyzer settings in Logic 1 for saving setups, saving captures, and for holding state when you close & re-open the software.

You will need to add a corresponding load call in `LoadSettings`, in the same order as here in SaveSettings (last)

Note - we'll never build this update for Logic 1 ourselves, so this would only affect users who downloaded and built this themselves for Logic 1, so it's pretty unlikely to happen.

If you see how there are 2 variables in the load code already wrapped in `if` statements, that's because the << operator for `SimpleArchive` always returns a boolean, true if the variable loaded, false if it did not. This is a big difference from a traditional stream. It was added to avoid throwing an exception if the saved settings failed to load for some reason.